src/utils.ts

```diff
@@ -8,3 +8,25 @@ export function envValueToBoolean(value: string | undefined | boolean, defaultVa
   }
   return defaultValue;
 }
+
+export function formatDateTime(value: string, language: string) {
```

Comment:
Would there be an easy way to handle a default case for the language? We should already have that information in the Redux store I guess, or somewhere else. So if a language would not be passed as a parameter, then we could just use what the user has set globally as their language of choice.
This would also remove the need for importing the `react-i18next` packages in a lot of components?

Reply:
I included the selected language in the helper function itself, so now there's no need to pass it every time we use that.